<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorsView</class>
 <widget class="QWidget" name="SensorsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,1">
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0">
    <widget class="Tools::MultiBarWidget" name="avoidingSensorsBars" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="Tools::MultiBarWidget" name="otherSensorBars" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Other sensors</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnPause">
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Avoiding sensors</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="Tools::MultiCurveWidget" name="otherSensorsPlot" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="Tools::MultiCurveWidget" name="avoidingSensorsPlot" native="true"/>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tools::MultiBarWidget</class>
   <extends>QWidget</extends>
   <header>NPlotWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tools::MultiCurveWidget</class>
   <extends>QWidget</extends>
   <header>NPlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
